### PR TITLE
TECH TASK: [#162110] New attempt at fixing Journal specs

### PR DESCRIPTION
**Pull request by LeticiaErrandonea**
# Release Notes
* TECH TASK: In Journal specs, click button again if it's still visible.

# Screenshot

Optional.

# Additional Context

Optional. Feel free to add/modify additional headers as appropriate, e.g. "Refactorings", "Concerns".

### CODE REVIEW

**Comment by LeticiaErrandonea** [comment]
@giladshanan built a couple of times and worked. I think this is ready to be reviewed. I'd like to keep the wait time as low as possible, so if this makes specs pass I wouldn't increase it

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -177,8 +177,13 @@
       WebMock.disable_net_connect!
     else
       WebMock.disable_net_connect!(allow_localhost: true)
+      old_default_max_wait_time = Capybara.default_max_wait_time
```
Comment (giladshanan) [inline spec/rails_helper.rb:180]: Please add a code comment explaining the reason for this change

**Review by giladshanan — APPROVED** [review]
🤞🏼 looks good, just left a small request

**Comment by LeticiaErrandonea** [comment]
@giladshanan given I've increased the wait time, but at least one spec still fail, I'll skip it (line 123). Let me know if you think I should skip more of them

File: spec/system/creating_a_journal_spec.rb [hunk 2]
```diff
@@ -127,10 +127,12 @@
         expect(page).to have_content("We are in the year-end closing window.")
         click_button "Create Journal"
         wait_for_ajax
+        # Sometimes the first click doesn't work, so try again
+        click_button "Create Journal" unless page.has_content?("Pending Journal")
```
Comment (giladshanan) [inline spec/system/creating_a_journal_spec.rb:131]: @LeticiaErrandonea what do you think of this idea?  It works for me locally, and I think I like it better than skipping the specs or sinking more time into debugging :) 

If you are not opposed to the approach, then:
1. Apply to the other flaky specs in this file
2. Test if we can remove the changes to rails_helper in this PR

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -177,8 +177,17 @@
       WebMock.disable_net_connect!
     else
       WebMock.disable_net_connect!(allow_localhost: true)
+      # Increase the default wait time for Capybara from 2 seconds to 5 seconds.
```
Comment (giladshanan) [inline spec/rails_helper.rb:180]: I think we can remove these changes and keep wait_for_ajax, since other specs are using that as well

Comment (LeticiaErrandonea) [reply spec/rails_helper.rb:180]: You're right

**Review by giladshanan — APPROVED** [review]

**Pull request merged**